src/main/java/net/pms/network/RequestV2.java

```diff
@@ -419,41 +461,56 @@ public ChannelFuture answer(
 						startStopListenerDelegate.start(dlna);
 
 						// Try to determine the content type of the file
-						String rendererMimeType = getRendererMimeType(dlna.mimeType(), mediaRenderer, dlna.getMedia());
 
 						if (rendererMimeType != null && !"".equals(rendererMimeType)) {
-							output.headers().set(HttpHeaders.Names.CONTENT_TYPE, rendererMimeType);
+							output.headers().set(HttpHeaderNames.CONTENT_TYPE, rendererMimeType);
+							if ("audio/L16".equals(rendererMimeType)) {
+								output.headers().set(
+										HttpHeaderNames.CONTENT_TYPE,
+										String.format("%s;channels=%d;rate=%d", rendererMimeType, 2,
+//												dlna.getMediaAudio().getAudioProperties().getNumberOfChannels(), 
+												dlna.getMediaAudio().getAudioProperties().getSampleFrequency()));
+							}
 						}
 
 						// Response generation:
 						// We use -1 for arithmetic convenience but don't send it as a value.
 						// If Content-Length < 0 we omit it, for Content-Range we use '*' to signify unspecified.
-						boolean chunked = mediaRenderer.isChunkedTransfer();
+						boolean chunked = true;//mediaRenderer.isChunkedTransfer();
```

Comment:
@SubJunk I've moved media streaming logic to RequesthandlerV2. Chunked mode is used only for transcoding scenario.